src/admin/style.scss

```diff
@@ -125,8 +121,8 @@ h1 {
 
 		// Specificity necessary to override @emotion/* styles
 		select.components-select-control__input {
-			background-color: var(--np-color-wp-gray-100);
-			border: 1px solid var(--np-color-wp-gray-100);
+			background-color: var(--newspack-ui-color-neutral-10);
+			border: 1px solid var(--newspack-ui-color-neutral-10);
```

Comment:
`wp-colors.$gray-100`?

Reply:
This was done in an attempt to keep things DRY'r by using `newspack-ui` derived variables. 
https://github.com/Automattic/newspack-plugin/blob/51cbc0ab26dcb7208762404c8b33a1906a35ce67/src/newspack-ui/scss/variables/_colors.scss#L5

Reply:
> Newspack UI colours are designed for the front-end.
This is a important distinction I did not know about. Thanks for this!

Have updated to use `wp-colors.$gray-100`